client/src/components/Tool/ToolSuccess.vue

```diff
@@ -1,42 +1,45 @@
 <script setup lang="ts">
 import { computed, onMounted } from "vue";
+import { useConfig } from "@/composables/config";
 import { useJobStore } from "@/stores/jobStore";
 import { useRouter } from "vue-router/composables";
-import { getGalaxyInstance } from "@/app";
 import ToolEntryPoints from "@/components/ToolEntryPoints/ToolEntryPoints.vue";
-import ToolSuccessMessage from "./ToolSuccessMessage.vue";
 import ToolRecommendation from "../ToolRecommendation.vue";
+import ToolSuccessMessage from "./ToolSuccessMessage.vue";
 import Webhook from "@/components/Common/Webhook.vue";
 
+const { config } = useConfig();
 const jobStore = useJobStore();
 const router = useRouter();
 
+const jobDef = computed(() => responseVal.value.jobDef);
+const jobResponse = computed(() => responseVal.value.jobResponse);
 const responseVal = computed(() => jobStore.getLatestResponse);
+const showRecommendation = computed(() => config.value.enable_tool_recommendations);
+const toolName = computed(() => responseVal.value.toolName);
 
 /* lifecyle */
 onMounted(() => {
-    // no response means no ToolForm ran in this session (nothing in store)
+    // no response means that no tool was run in this session i.e. no data in the store
     if (Object.keys(responseVal.value).length === 0) {
         router.push(`/`);
     }
 });
-
-const jobDef = computed(() => responseVal.value.jobDef);
-const jobResponse = computed(() => responseVal.value.jobResponse);
-const toolName = computed(() => responseVal.value.toolName);
-
-// getGalaxyInstance is not reactive
-const configEnableRecommendations = getGalaxyInstance().config.enable_tool_recommendations;
-const showRecommendation: boolean = [true, "true"].includes(configEnableRecommendations);
 </script>
 
 <template>
     <section v-if="jobResponse && jobDef">
         <div v-if="jobResponse.produces_entry_points">
             <ToolEntryPoints v-for="job in jobResponse.jobs" :key="job.id" :job-id="job.id" />
         </div>
-        <ToolSuccessMessage :job-def="jobDef" :job-response="jobResponse" :tool-name="toolName" />
+        <ToolSuccessMessage :job-response="jobResponse" :tool-name="toolName" />
         <Webhook type="tool" :tool-id="jobDef.tool_id" />
         <ToolRecommendation v-if="showRecommendation" :tool-id="jobDef.tool_id" />
     </section>
+    <section v-else>
+        <b-alert v-localize show variant="info">
+            The last job details are not available anymore, but you can obtain this information by accessing the
```

Comment:
I don't think I would understand what that means, and when this would be the case ... is it necessary ?

Reply:
I added this as a fallback since the previous statement also specifies an explicit if-condition, but that statement is likely also unnecessary. Removed the alert.